Contents/Code/support/items.py

```diff
@@ -67,6 +70,10 @@ def getItems(key="recently_added", base="library", value=None, flat=False, add_s
         else:
             kind = item.type
 
+        if item.thumb:
```

Comment:
Why do you set a global variable here? Isn't setting the thumb in menu_helpers:dig_tree enough?